doc/flash-manipulation.md

```diff
@@ -0,0 +1,136 @@
+# SMC flash manipulation tools, v6
+Author: Eirik Stople
+
+The programs found in the [tools](../tools) folder allow X16 to dump entire SMC flash, and manipulate bootloader.
+
+
+## New I2C commands
+
+New I2C commands to SMC application, added in SMC version 47.2.0:
+
+- Write $90: set page to read/write
+- Read  $91: Read byte from flash and increment pointer
+- Write $92: Write byte to flash buffer and increment pointer
+    - Will write to flash once a full page (64 bytes) has been received
+    - Will only manipulate flash in bootloader area (page 120-127)
+- Read  $93: Check if flash manipulation is unlocked (0: locked, 1: unlocked, 255: old SMC version)
+- Write $93: Request flash manipulation (1: flash manipulation unlock request, 0: flash manipulation lock)
+
+
+## Usage
+
+1. Install SMC version 47.2.0 or later
+- Follow the guide "how to use the bad version of bootloader v2"
+- Optionally, use an ISP programmer/Arduino
+- Use the HELP command and verify that you have version 47.2.0 or later before you proceed.
+
+2. Copy tools/SMCBLD2.PRG and tools/SMCBLW16.PRG from https://github.com/X16Community/x16-smc to the SD card
+
+3. Copy bootloader.bin from https://github.com/X16Community/x16-smc-bootloader/releases/tag/v3 to the SD card
```

Comment:
My original instructions suggests updating to the non-corrupt version 2. If we should suggest users to install v3 instead, I suggest to update the tools and the instructions, so that the v3 checksum is recognized. If not, a corrupt file is not detected on the X16, which it can if the checksum is embedded